wallet/wallet-rpc-lib/src/rpc/types.rs

```diff
@@ -104,6 +106,12 @@ pub enum RpcError<N: NodeInterface> {
 
     #[error("Invalid hex data deposit")]
     InvalidHexData,
+
+    #[error("Provided addresses to add a Multisig are not all public keys")]
```

Comment:
Provided addresses to create a Multisig address are not all public keys. Only public keys can be used.